### PR TITLE
Release 0.4.0

### PR DESCRIPTION
**Pull request by Marianasilvago**
### What

- Upgrade to log.go  v2 version
- Migrate to official mongo-go-driver for supporting document db.

### How to review

Check only required commits are going in.

### Who can review

Anyone

### CODE REVIEW

File: mongo/mongo.go [hunk 1]
```diff
@@ -55,10 +71,10 @@ func (m *Mongo) Init(mongoConf config.MongoConfig) error {
 
 // Close closes the mongo session and returns any error
 func (m *Mongo) Close(ctx context.Context) error {
-	if m.Session == nil {
-		return errors.New("cannot close a mongoDB connection without a valid session")
+	if m.Connection == nil {
+		return errors.New("cannot close a empty connection")
 	}
```
Comment (HeJones) [inline mongo/mongo.go:76]: a tiny typo, an rather than a

**Review by HeJones — APPROVED** [review]
It looks good to me

**Review by HeJones — APPROVED** [review]
The changes looks good to me

**Review by morses-code — APPROVED** [review]
LGTM

**Pull request closed without merging**